spring-integration-core/src/main/java/org/springframework/integration/endpoint/AbstractPollingEndpoint.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2002-2016 the original author or authors.
+ * Copyright 2017 the original author or authors.
```

Comment:
No, you have to leave the first year and fix the last to the current.
The first year is about the birth date, the last about maturity.